alphabet/bag_test.go

```diff
@@ -124,3 +125,20 @@ func TestRemoveTiles(t *testing.T) {
 	is.NoErr(err)
 	is.Equal(len(bag.tiles), 9)
 }
+
+func TestFixedOrder(t *testing.T) {
+	is := is.New(t)
+
+	ld, err := EnglishLetterDistribution(&DefaultConfig)
+	is.NoErr(err)
+	bag := NewBag(ld, ld.alph)
+	sort.Slice(bag.tiles, func(i, j int) bool { return bag.tiles[i] > bag.tiles[j] })
+
+	bag.SetFixedOrder(true)
+	is.Equal(len(bag.tiles), 100)
+	ml := make([]MachineLetter, 17)
+	err = bag.Draw(17, ml)
+	is.NoErr(err)
+
+	is.Equal(ml, []MachineLetter{3, 3, 3, 3, 2, 2, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0})
```

Comment:
any idea why it's drawing 4 D, 2 C, 2 B, 9 A, instead of from the back (3 T, 4 U, 2 V, 2 W, 1 X, 2 Y, 1 Z, 2 ?)?

Reply:
my sort function right above flips the lessthan condition